package.json

```diff
@@ -7,7 +7,7 @@
     "start": "npm run grunt -- default",
     "build": "npm run prod",
     "prod": "npm run grunt -- buildProd",
-    "grunt": "./node_modules/grunt-cli/bin/grunt",
+    "grunt": "node ./node_modules/grunt-cli/bin/grunt",
     "lint": "./node_modules/eslint/bin/eslint.js .",
```

Comment:
I guess we can add `node` here too

Reply:
That change is what I'm requesting, it's the only line I've already changed.

Reply:
you changed the line for `grunt` command,  I'm talking about the `lint` command.